components/playlist/resources/components/app.tsx

```diff
@@ -198,7 +210,7 @@ export class PlaylistPage extends React.Component<Props, State> {
           </Table>
         </div>
 
-        <video id="player" controls autoPlay />
+        <iframe id="player" src="chrome-untrusted://playlist-player" style={{ width: '512px', height: '288px', border: 'none' }} allow="autoplay" scrolling='no'/>
```

Comment:
is it possible to set the sandbox attribute here and allow values as needed? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#attr-sandbox

Reply:
Cool! Done.